about.html

```diff
@@ -0,0 +1,126 @@
+                            <li><a class="mobile-link" href="home.html">Home</a></li>
+                            <li><a class="mobile-link" href="about.html">About</a></li>
+                        </ul>
+                    </div>
+
+                    <div class="hamburger">
+                        <div class="bar"></div>
+                        <div class="bar"></div>
+                        <div class="bar"></div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </header>
+    <section>
+        <div class="headline-container">
+            <p class="headline-p">"Hello! scholar"</p>
+            <h1>Educational Resources Summit 2023</h1>
+            <p class="headline-summary">
+                Educational resources are used in a learning environment to help and assist with people’s development
+                and
+                learning. We are designed to reinforce learning and in some cases allow people to put their knowledge to
+                the
+                test. Educational resources are brilliant for teachers and educators to help them deliver the best
+                quality
+                lessons.
+            </p>
+            <P class="venue">
+                Please contacts us via Email for any further questions about educational resources 2023!
+                <a href="" class="link">educationalresources@edu.ng</a>
+            </P>
+        </div>
+        <div class="theme">
+            <h2>Theme</h2>
+            <p class="theme-p">
+                "Success occurs when opportunity meets preparation".
+            </p>
+            <p class="summit-logo">
+                <i class="fa-solid fa-graduation-cap cap"></i>
+                <a class="logo-link" href="">Educational Resources</a>
+            </p>
+        </div>
+    </section>
+
+    <hr class="line">
+    <section class="past-summit">
+        <h2>See the past Educational Resources summit</h2>
+        <p class="past-summit-p">
+            Take a look at the last educational resources summit which took place in Lagos, Nigeria
+        </p>
+        <div class="enbeded-videos">
+            <div class="video">
+                <iframe width="350" height="200" src="https://www.youtube.com/embed/CeWA4USaie8?start=96"
+                    title="YouTube video player" frameborder="0"
+                    allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+                    allowfullscreen></iframe>
+            </div>
+            <div class="video">
+                <iframe width="350" height="200" src="https://www.youtube.com/embed/2gYCgwtVhL0?start=96"
+                    title="YouTube video player" frameborder="0"
+                    allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+                    allowfullscreen></iframe>
+            </div>
+        </div>
+    </section>
```

Comment:
- [ ] What do you think if you use images in this section instead of videos as in the template design? Also, make sure you match its design to the one from the template by adding all the possible elements(e.g.: background on images, texts,...). This will make you achieve the expected result as matching the given design is one of the objectives of the project.

| Template Design | Your Design |
| -------------------- | ---------------------- |
| ![Screenshot from 2023-01-14 21-53-04](https://user-images.githubusercontent.com/83042742/212493798-5977b711-813a-4d68-8d0e-acddd67c73d8.png) | ![Screenshot from 2023-01-14 21-50-47](https://user-images.githubusercontent.com/83042742/212493806-fdca1122-70ab-4e08-9ffa-dd1b2bd7b883.png) |